src/styles/globals.css

```diff
@@ -10,6 +10,7 @@ body {
   background-color: var(--white);
   padding: 0;
   margin: 0;
+  font-size: 16px;
```

Comment:
Setting `font-size` here lets us use `rem` elsewhere. Relative `em`s are generally the best way to size stuff.